rcon/extended_commands.py

```diff
@@ -463,6 +461,16 @@ def set_maprotation(self, rotation):
 
         return [first] + rotation
 
+    def do_perma_ban_and_return_ban_log(self, player, reason):
```

Comment:
It's a shame this MR is entangled with the ban one. I'll resume the review once we merge the ban one